### PR TITLE
[ready] set utm_content from id or text of a tag

### PR DESCRIPTION
**Pull request by longneter**
Keep a commit from https://github.com/nbphuoc/ahoy_email
And set `utm_content` from id or text of `a` tag to check where the visit from

A part of [#BUG1-202 Là MKT, tôi muốn có các link campaign khác nhau để phân loại việc track mở email với việc nhấn vào các link trong email](https://app.vivifyscrum.com/boards/88480/sprint-backlog/280145/BUG1-202)

### CODE REVIEW

**Comment by longneter** [comment]
`master` nó drop support rails 4.2 chắc em rebase về trước commit này quá https://github.com/ankane/ahoy_email/commit/cab469377358b59a7b191b66b7402ce909b4f547

File: test/internal/app/mailers/utm_params_mailer.rb [hunk 1]
```diff
@@ -12,4 +12,8 @@ def basic
   def array_params
     mail_html('<a href="https://example.org?baz[]=1&amp;baz[]=2">Hi<a>')
   end
+
+  def has_id
```
Comment (tiendo1011) [inline test/internal/app/mailers/utm_params_mailer.rb]: This should be named id_or_text, that's how the above method is named.

File: test/utm_params_test.rb [hunk 2]
```diff
@@ -23,4 +23,11 @@ def test_array_params
     message = UtmParamsMailer.array_params.deliver_now
     assert_body "baz%5B%5D=1&amp;baz%5B%5D=2", message
   end
+
+  def test_utm_content
```
Comment (tiendo1011) [inline test/utm_params_test.rb]: Same here, should be named `test_id_or_text`.

**Comment by tiendo1011** [comment]
Just 2 minor fixes, then you can mark this as ready yourself.

**Comment by longneter** [comment]
don't delete this branch: `get-utm-content-by-id-or-text-of-a`

**Pull request merged**